CHANGES.md

```diff
@@ -1,5 +1,18 @@
 # Changes
 
+## edge-21.4.3
+
+This edge supersedes `edge-21.4.2` as a release candidate for `stable-2.10.1`!
+
+This release adds support for TrafficSplit `v1alpha2`. `v1alpha1` will be
+marked deprecated and removed in a future release.
+Additionally, It includes improvements to the web and `proxy-init` images.
```

Comment:
nit: Can "Additionally" either continue on the line above, or a have a newline above it? It looks a little weird right now as it is currently formatted.